source/Octopus.Tentacle.Tests.Integration/TentacleCommandLineTests.cs

```diff
@@ -119,7 +118,7 @@ public async Task VersionCommandTextFormat(TentacleConfigurationTestCase tc)
 
             var expectedVersion = GetVersionInfo(tc);
 
-            stdout.Should().Be(expectedVersion.ProductVersion, "The version command should print the informational version as text");
+            stdout.TrimEnd().Should().Be(expectedVersion.ProductVersion, "The version command should print the informational version as text");
```

Comment:
CliWrap deals with text output in a slightly different way to Shellfish.

If the app emits text without a trailing newline, CliWrap wouldn't give you a trailing newline either, whereas with Shellfish we get a trailing newline.

This isn't an inherent flaw in Shellfish, but rather just a side effect `WithStdOutTarget(stringBuilder)` output mechanism. The stringbuilder-writer puts a newline on the end of everything to keep things simple. We could do `WithStdOutTarget(someCustomTarget)` and get the output in a more direct way, but that would be harder and more complex than simply trimming the newlines in a few places

Reply:
For the test I don't think it matters.